src/omero_cli_render.py

```diff
@@ -520,23 +520,23 @@ def set(self, args):
                 cindices.append(-i)
             else:
                 cindices.append(i)
-            rangelist.append([c.min, c.max])
+            rangelist.append([c.start, c.end])
```

Comment:
Addtionally checking here for `c.min` and `c.max` here, could make the changes of this PR backward compatible. The question then is, what should be done if min/max and start/end are specified:
- min/max overrides start/end
- start/end take precedence
- throw an error

Reply:
I think we will have to handle previous versions of the YML files especially since we extensively used this format in IDR.

My understanding is that below are the proposed behaviors for the two format versions:

- version 1 (current): channel `min/max` sets the channel window start/end, channel `start/end` ignored (or maybe simply not part of the render YML/JSON specification?)
- version 2 (current): channel`start/end` sets the channel window start/end, `min/max` is now either a no-op or not present for the YML/JSON specification

One way we might consider to make this unambiguous would be to start introduce a `version: 2` field in the YAML/JSON specification. The question now remains: what would happen when such `version` field is absent? Options are:

- always default to `version: 1`. This means that a YML file with `start/end` _without_ `version: 2` would probably not behave as expeced.
- as suggested above, detect and error if `start/end` and `min/max` are both found in the format. A potential implication is that a YML file with only `start/end` and no `version` should be auto-detected as version 2 while a YML file with only `min/max` and no `version` would be autodetected as version 1.

Reply:
I'd vote for the last option. Add a version field. If it's not specified start/end implies version 2, min/max implies version 1, if both specified error.